docs/zkapps/tutorials/03-deploying-to-a-network.mdx

```diff
@@ -114,15 +117,17 @@ Start by deleting the default files that come with the new project.
 
 Now that your smart contract is in place, you are ready to deploy your smart contract to Berkeley Testnet.
 
-## Mina zkApp CLI
+## zkApp CLI
 
-You already installed the Mina zkApp CLI as part of the [Prerequisites](/zkapps/tutorials#prerequisites), so you have the tools to manage deployments.
+You installed the zkApp CLI as part of the [Prerequisites](/zkapps/tutorials#prerequisites), so you already have the tools to manage deployments.
 
 In some cases, you might need to create a custom account for your zkApp, for example, to deploy a zkApp to a different key than the fee payer key, programmatically parameterize a zkApp before you initialize it, or create a smart contract programmatically for users as part of an application. For details, see [Interacting with zkApps server-side](/zkapps/tutorials/interacting-with-zkapps-server-side).
```

Comment:
Reviewer: what are we saying here? is this why we have the server-side tuto?

Reply:
What we are describing is a way to write programs that actually use our deployed zkApps (or do other things in that vein). There are many situations where this is useful; basically, anytime you want something done, it's impossible/unpleasant to do it manually.